jobs/JGLOBAL_ATMOS_UPP

```diff
@@ -12,8 +12,8 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "upp" -c "base upp"
 ##############################################
 
 # Construct COM variables from templates
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS COM_ATMOS_HISTORY COM_ATMOS_MASTER
-if [[ ! -d ${COM_ATMOS_MASTER} ]]; then mkdir -m 775 -p "${COM_ATMOS_MASTER}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMIN_ATMOS_ANALYSIS COMIN_ATMOS_HISTORY COMOUT_ATMOS_MASTER
```

Comment:
Because of the way `declare_from_tmpl` works, once you change `COM` to `COMIN` or `COMOUT`, you have to explicitly specify the template to use (when not provided, `declare_from_tmpl` assume the template name is `<variable>_TMPL`, which is no longer true when using `COMIN`/`COMOUT`)
```suggestion
YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
    COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
    COMIN_ATMOS_HISTORY:COM_ATMOS_HISTORY \
    COMOUT_ATMOS_MASTER:COM_ATMOS_MASTER
```